### ISSUE TITLE
A `%` sign is added at the end of `dapr version` command output

### ISSUE THREAD

**Issue by mukundansundar** [dapr/cli#1013]
## Expected Behavior

No `%` sign is added.

## Actual Behavior

A `%` sign is added at the end of `dapr version` command output

```bash
$ dapr version               
CLI version: edge 
Runtime version: 1.5.3%          
```
```bash
$ dapr version --output json 
{"Cli version":"edge","Runtime version":"1.5.3"}%    
```

## Steps to Reproduce the Problem

Build latest version of code from release-1.8 branch and run the `dapr version` command.

## Release Note

RELEASE NOTE:

**Comment by shivamkm07** [comment]
Checked with rc1 release as well as release 1.8 build on linux: works fine, no % sign added in version

**Comment by pravinpushkar** [comment]
Works fine on windows too.

**Comment by mukundansundar** [comment]
In Mac in ZSH, ZSH automatically added a new line at the end of output and added a highlighted `%` sign at the previous line to potentially indicate the missing newline char. In bash, it works as expected without any issue.

**Comment by mukundansundar** [comment]
closed by #1014
